cvat/apps/engine/mixins.py

```diff
@@ -247,9 +255,9 @@ def upload_data(self, request):
         else: # backward compatibility case - no upload headers were found
             return self.upload_finished(request)
 
-    def init_tus_upload(self, request):
+    def init_tus_upload(self, request: PatchedRequest):
         if request.method == 'OPTIONS':
-            return self._tus_response(status=status.HTTP_204)
+            return self._tus_response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
That's interesting... was this broken before? If so, you should probably add a changelog-entry, as this is a user-visible change.

Reply:
yes, it was
![image](https://github.com/user-attachments/assets/3f80f437-a155-4051-b58f-aa95cd3312cc)

Reply:
Mind adding the changelog entry, then?